demo/app/main-page.ts

```diff
@@ -1,15 +1,16 @@
 import { EventData } from "tns-core-modules/data/observable";
 import { Button } from "tns-core-modules/ui/button";
+import { topmost } from "tns-core-modules/ui/frame"
```

Comment:
This line is missing a semicolon, which is breaking the travis CI build.

Also, "Button" no longer needs to be imported.